nodeup/pkg/model/protokube.go

```diff
@@ -267,6 +268,11 @@ func (t *ProtokubeBuilder) ProtokubeFlags(k8sVersion semver.Version) (*Protokube
 		f.TLSCAFile = s(filepath.Join(t.PathSrvKubernetes(), "ca.crt"))
 		f.TLSCertFile = s(filepath.Join(t.PathSrvKubernetes(), "etcd.pem"))
 		f.TLSKeyFile = s(filepath.Join(t.PathSrvKubernetes(), "etcd-key.pem"))
+
+		enableAuth := true
+		if t.UseTLSAuth() {
```

Comment:
As this is the only place you use `UseTLSAuth`, maybe just inline it here - as you say, it's an odd function, and it's only because of limitations in how we communicate with protokube...

Reply:
it's nice to keep purely for consistency with UseEtcdTLS() and moving code out of the logic section .. but i'm happy to move back in?